commands/test.go

```diff
@@ -114,17 +114,23 @@ func (c TestCommand) Execute() int {
 	reportDir = path.Join(wd, reportDir)
 	err = os.Mkdir(reportDir, 0755)
 	if err != nil {
-		log.Fatalf("[Error] error creating report dir %s: %+v", reportDir, err)
+		log.Fatalf("[ERROR] error creating report dir %s: %+v", reportDir, err)
 	}
 
 	if applyErr == nil && len(tf.GetChanges(plan)) == 0 {
 		if state, err := terraform.Show(); err == nil {
 			passReport := tf.NewPassReportFromState(state)
-			storePassReport(passReport, reportDir, "all_passed_report.md")
 			log.Printf("[INFO] %d resources passed the tests.", len(passReport.Resources))
+
+			log.Printf("[INFO] producing coverage report...")
```

Comment:
>log.Printf("[INFO] producing coverage report...") [](http://example.com/codeflow?start=3&length=49)

Suggest order:

tf.NewCoverageReportFromState(state)

log.Printf("[INFO] the coverage report has been produced")

storePassReport

log.Printf("[INFO] all reports have been saved in the report directory: %s, please check.", reportDir)

Reply:
thanks, I have changed the code.